.github/workflows/test.yml

```diff
@@ -100,6 +96,12 @@ jobs:
           wait-on: |
             http://localhost:3000/graphql
 
+      - name: Kysely Codegen
+        run: yarn pg:generate
+
+      - name: Check Code Quality
+        run: yarn codecheck
```

Comment:
i added type safety on the embedding model tables so we can't code check until the embedding service starts up